python/src/ngraph/ops.py

```diff
@@ -2279,14 +2375,14 @@ def sigmoid(data):  # type: (Node) -> Node
 
 
 @nameable_op
-def shape_of(data):  # type: (Node) -> Node
+def shape_of(data, output_type='i64'):  # type: (Node, str) -> Node
     """Return a node which produces a tensor containing the shape of its input data.
 
-    :param data: The tensor containing the input data
+    :param data: The tensor containing the input data.
+    :para output_type: Output element type.
     :return: ShapeOf node
     """
-    # TODO: update to use opset3 by default
-    return _get_node_factory('opset1').create('ShapeOf', [data])
+    return _get_node_factory('opset3').create('ShapeOf', [data], {'output_type': output_type})
```

Comment:
We're already switching to `opset3` by default, so you can remove the explicit call here.

Reply:
Done